spotbugs/src/main/java/edu/umd/cs/findbugs/detect/MethodCallInOptionalOrElse.java

```diff
@@ -0,0 +1,150 @@
+package edu.umd.cs.findbugs.detect;
+
+import java.util.Arrays;
+
+import edu.umd.cs.findbugs.ba.XField;
+import edu.umd.cs.findbugs.ba.bcp.Opcode;
```

Comment:
The build is failing due to this needless `import`. Please run `./gradlew spotlessApply` to format your code.

Reply:
done